LocalMultiplayerAgent/readme.md

```diff
@@ -18,3 +18,4 @@ Once the files are copied over, the LocalMultiplayerAgent file will need to be c
 ## MultiplayerSettings.json
 
 The [MultiplayerSettings.json generator tool](./SettingsJsonGenerator/README.md) can assist you with configuring your MultiplayerSettings.json file, and configuring it for your server.
+The [DeploymentSettings.json](./DeploymentTool/readme.md) can assist you with configuring your DeploymentSettings.json file for running deployment on LMA.
```

Comment:
recommendation - favor not using acronyms in documentation if you can help it - it helps with searching and understanding if you spell it out.